### PR TITLE
Allow change regex patterns and ignore tags with option param.

### PR DESCRIPTION
**Pull request by saiddjelmo**
Allow change regex patterns and ignore tags with option param.

Example:
```
this.editor = new Quill('#editor', options);

const markdownOptions = {
  ignoreTags: ['H1', 'H2'],
  tags: {
    bold: {
      pattern: /(\*){1}(.+?)(?:\1){1}/g,
    },
    italic: {
      pattern: /(\_){1}(.+?)(?:\1){1}/g,
    },
  },
};
const markdown = new QuillMarkdown(this.editor, markdownOptions);
```

### CODE REVIEW

**Comment by cloverhearts** [comment]
I'll look at on today

File: src/tags/strikethrough/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Bold {
-  constructor (quillJS) {
+  constructor (quillJS, options = {}) {
     this.quillJS = quillJS
     this.name = 'strikethrough'
-    this.pattern = /(?:~|_){2}(.+?)(?:~|_){2}/g
+    this.pattern =  options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g
```
Comment (cloverhearts) [inline src/tags/strikethrough/index.js:5]: If you want to pass test case, you need to replace multiple white space to single white space.

Mean:
```
this.pattern =..options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g

to

this.pattern =.options.pattern || /(?:~|_){2}(.+?)(?:~|_){2}/g

```

Dot is whitespace mean

**Review by cloverhearts — CHANGES_REQUESTED** [review]
Thank you for your NICE PR.
It was a Nice approach!

Can you fix a code style issue?
if so i'll merge to quilljs-markdown!

File: src/tags/strikethrough/index.js (hunk 1 above)
Comment (saiddjelmo) [reply src/tags/strikethrough/index.js]: Thank you. I fixed it.

Comment (saiddjelmo) [reply src/tags/strikethrough/index.js]: Thank you. I fixed it.

Comment (cloverhearts) [reply src/tags/strikethrough/index.js]: AWESOME!

**Review by cloverhearts — APPROVED** [review]
Ready for merge

**Pull request merged**